libcodechecker/analyze/analysis_manager.py

```diff
@@ -77,6 +80,50 @@ def worker_result_handler(results, metadata, output_path):
 
     metadata['result_source_files'] = source_map
 
+
+def create_dependencies(action):
+    """
+    Transforms the given original build 'command' to a command that, when
+    executed, is able to generate a dependency list.
+    """
+    if 'CC_LOGGER_GCC_LIKE' not in os.environ:
+        os.environ['CC_LOGGER_GCC_LIKE'] = 'gcc:g++:clang:clang++:cc:c++'
```

Comment:
In case this needs to be set for the log command, maybe it is worth to have a warning for the user? Otherwise looks good.